java/src/jmri/jmrix/dccpp/DCCppTurnoutManager.java

```diff
@@ -152,32 +152,33 @@ public boolean allowMultipleAdditions(String systemName) {
 
     /**
      * Get the bit address from the system name.
+     * @return -1 for failure
      */
     public int getBitFromSystemName(String systemName) {
         // validate the system Name leader characters
         if ((!systemName.startsWith(getSystemPrefix() + typeLetter()))) {
             // here if an illegal DCC++ turnout system name
             log.error("illegal character in header field of DCC++ turnout system name: {} prefix {} type {}",
                     systemName, getSystemPrefix(), typeLetter());
-            return (0);
+            return -1;
         }
         // name must be in the DCCppTnnnnn format (DCCPP is user configurable)
         int num = 0;
         try {
             num = Integer.valueOf(systemName.substring(
                     getSystemPrefix().length() + 1, systemName.length())).intValue();
         } catch (Exception e) {
-            log.debug("invalid character in number field of system name: {}", systemName);
-            return (0);
+            log.error("invalid character in number field of system name: {}", systemName);
```

Comment:
This will make the "Add new turnout" pane validation rather noisy while typing, as ``validSystemNameFormat("1")`` invokes ``getBitFromSystemName("1")`` on every keystroke. So for (another type of) consistency I suggest to leave this series at debug level.